### ISSUE TITLE
reverse dns

### ISSUE THREAD

**Issue by HappyBasher** [linuxmuster/sophomorix4#5]
sophomorix-devices has to create reverse dns entries for devices. Linbo needs this to function properly.
Example:
- create zone for 10.16.100:
  `samba-tool dns zonecreate localhost 100.16.10.in-addr.arpa`
- create reverse dns entry for host 10.16.100.1:
  `samba-tool dns add localhost 100.16.10.in-addr.arpa 1 PTR r100-pc01.linuxmuster.local`

**Comment by jeffbeck** [comment]
How many dns zones do we need? One? Then the following creates this zone
samba-tool dns zonecreate localhost 10.in-addr.arpa

and the following adds a host
samba-tool dns add localhost 10.in-addr.arpa 1 PTR r100-pc01.linuxmuster.local

**Comment by jeffbeck** [comment]
Reverse DNS works in 3.9.1-1

**Comment by HappyBasher** [comment]
DNS reverse lookup should return the fqdn. In fact it returns only the host part:
```
root@server:~# nslookup 10.0.0.254
254.0.0.10.in-addr.arpa name = firewall.
```
Should be:
```
254.0.0.10.in-addr.arpa name = firewall.linuxmuster.lan.
```
So the DNS entry is somewhat incomplete. This issue is critical for web-proxy authentication.

**Comment by jeffbeck** [comment]
This is fixed in 3.60.24-1

Please confirm, that is does what you want.

Unfortunately the added `.linuxmuster.local` is not stored in the cn attribute. Only in the binary dns data.

So, there is no way to update the reverse records besides: 
1) use empty devices.csv to clear all devices and add them again
    --> This will delete the computer password hashes as well
2) use an ldap browser to delete all entries with
      A) sophomorixDnsNodetype=reverse
      B) sophomorixDnsNodetype=lookup
     
I hope this can be handled on alpha installs??

See also https://github.com/linuxmuster/sophomorix4/issues/61

**Comment by HappyBasher** [comment]
Fix confirmed. Deletion and recreation of hosts should be ok. If there is a linbo client image the computer account password hash is restored on linbo sync.
